piezo_web_app/PiezoWebApp/src/handlers/job_status.py

```diff
@@ -0,0 +1,28 @@
+from tornado_json import schema
+
+from PiezoWebApp.src.handlers.base_handler import BaseHandler
+from PiezoWebApp.src.handlers.schema.schema_helpers import create_object_schema_with_string_properties
+
+
+# pylint: disable=abstract-method
+class JobStatusHandler(BaseHandler):
+    @schema.validate(
+        input_schema=create_object_schema_with_string_properties(
+            ['spark_job', 'namespace'], required=['spark_job', 'namespace']),
```

Comment:
It's the same as `job_name` from `deletejob`. Do you have a preference in which we stick with?

Reply:
I think `job_name` is clearer, as `spark_job` could be easily mixed up with `job_name`.

However, we could consider making it consistently `spark_job` and have the `getlogs` process append the `-driver` to make it what was the `driver_name`. Would this work?

Reply:
I think it would be best to have it consistent throughout and append `-driver` automatically when getting logs.  `job_name` probably would be best for this as it then is clearer in `submit_job` too

Reply:
eda47b5

19781a2